calico_containers/pycalico/datastore.py

```diff
@@ -25,10 +25,22 @@
 from pycalico.datastore_errors import DataStoreError, \
     ProfileNotInEndpoint, ProfileAlreadyInEndpoint, MultipleEndpointsMatch
 from pycalico.util import get_hostname
+import logging
+
+# Suppress warnings from urllib3 when etcd is used without CA cert.
+logging.captureWarnings(True)
```

Comment:
Actually i think this should be in calicoctl rather than here.  Also feel free to add logs to your branches, but you'll need to add a null handler for this logger.